operations/mimir-mixin/dashboards/block-builder.libsonnet

```
@@ -0,0 +1,95 @@
+local filename = 'mimir-block-builder.json';
+
+(import 'dashboard-utils.libsonnet') +
+{
+  [filename]:
+    assert std.md5(filename) == 'c565e768420d79d0a632b3135d47cb30' : 'UID of the dashboard has changed, please update references to dashboard.';
+    ($.dashboard('Block-builder') + { uid: std.md5(filename) })
+    .addClusterSelectorTemplates()
+    .addRow(
+      $.row('Summary')
+      .addPanel(
+        $.timeseriesPanel('Overall consumption / sec') +
+        $.panelDescription(
+          'Overall consumption / sec',
+          'Overview of per-second rate of records consumed from Kafka.',
```

This is fetched, not necessarily successfully processed, right?

```suggestion
          'Overview of per-second rate of records fetched from Kafka.',
```